internal/v2/s2av2_e2e_test.go

```diff
@@ -154,11 +165,21 @@ func TestEndToEndUsingFakeS2AOnUDS(t *testing.T) {
 	grpclog.Infof("fake S2A for client listening on UDS at address: %v", clientS2AAddr)
 
 	// Start the server.
-	serverAddr := startServer(t, serverS2AAddr)
+	var localIdentities []*commonpbv1.Identity
```

Comment:
For each unit test, can we actually have 2 unit tests: one with no local identity set, one with the local identity set?

Reply:
Will do! I separated all the E2E tests, so that in one, both the client and server set localIdentity, localIdentities, and in the other, neither client or server set localIdentity, localIdentities.